### PR TITLE
Rework the completer module.

### PR DESCRIPTION
**Pull request by knightburton**
- Apply the new eslint rules to the module.
- Rework the class definition based on ES6.

IoT.jsCode-DCO-1.0-Signed-off-by: Imre Kiss kissi.szeged@partner.samsung.com

### CODE REVIEW

File: src/app/completer.js [hunk 1]
```diff
@@ -13,249 +13,246 @@
...
-  { "word": "writeUInt8", "score": 300, "meta": "Buffer" },
-  { "word": "writeUInt16LE", "score": 300, "meta": "Buffer" },
-  { "word": "writeUInt32LE", "score": 300, "meta": "Buffer" },
-  { "word": "readInt8", "score": 300, "meta": "Buffer" },
-  { "word": "readUInt8", "score": 300, "meta": "Buffer" },
-  { "word": "readUInt16LE", "score": 300, "meta": "Buffer" },
+  /**
+   * IoT.js Basic and Extended API functions list.
+   */
+  get iotjsFunctions() {
```
Comment (daeyeon) [inline src/app/completer.js]: IMO, Completer seems a common utility. So the below list seems to be needed to define outside.

Comment (knightburton) [reply src/app/completer.js]: You are right, I will export this list outside the class.

**Comment by knightburton** [comment]
@daeyeon I've updated the PR based on your suggest, review it again please.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**